saleor/graphql/product/tests/test_product.py

```diff
@@ -2342,6 +2346,11 @@ def test_create_product(
     assert slugify(non_existent_attr_value) in values
     assert color_value_slug in values
 
+    product = Product.objects.first()
+    created_webhook_mock.assert_called_once_with(product)
+
+    assert not updated_webhook_mock.called
```

Comment:
You can use `assert_not_called` ([docs](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_not_called)):
```suggestion
    updated_webhook_mock.assert_not_called()
```